src/components/SearchForm.tsx

```diff
@@ -11,20 +12,20 @@ const SearchForm = (props: { query?: string }) => {
 		textarea.style.height = `${textarea.scrollHeight}px`
 		setTextareaInput(textarea.value)
 	}
-
+	const onSubmit = (e: Event) => {
+		e.preventDefault()
+		const textAreaValue = textareaInput()
+		const searchQuery = encodeURIComponent(
+			textAreaValue.length > 3800 ? textAreaValue.slice(0, 3800) : textAreaValue
+		)
+		const filters = selectedComboboxItems()
+			.map((item) => item.eventId)
+			.join(',')
```

Comment:
Is there a reason not to use encodeUriComponent here?

Reply:
No I just forgot to put that in, but it shouldn't be necessary as all the filters shouldn't require any special characters.